<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEarth</class>
 <widget class="QWidget" name="FormEarth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>540</height>
   </rect>
  </property>
  <widget class="QWidget" name="widgetControls" native="true">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>10</y>
     <width>131</width>
     <height>129</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QToolButton" name="toolButtonZoomDefault">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="toolButtonZoomOut">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="toolButtonMoveRight">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="toolButtonZoomIn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="toolButtonMoveUp">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="toolButtonMoveDown">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="toolButtonMoveLeft">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="toolButtonMoveDebug">
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../../Lib/CoreUi/Ui.qrc">
     <normaloff>:/Icons/ZoomIn.png</normaloff>:/Icons/ZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Приблизить</string>
   </property>
   <property name="toolTip">
    <string>Приблизить</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../../Lib/CoreUi/Ui.qrc">
     <normaloff>:/Icons/ZoomOut.png</normaloff>:/Icons/ZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Отдалить</string>
   </property>
   <property name="toolTip">
    <string>Отдалить</string>
   </property>
  </action>
  <action name="actionZoomDefault">
   <property name="icon">
    <iconset resource="LibZ.qrc">
     <normaloff>:/Icons/ZoomNone.png</normaloff>:/Icons/ZoomNone.png</iconset>
   </property>
   <property name="text">
    <string>Стандартный масштаб</string>
   </property>
   <property name="toolTip">
    <string>Стандартный масштаб</string>
   </property>
  </action>
  <action name="actionMoveLeft">
   <property name="icon">
    <iconset resource="LibZ.qrc">
     <normaloff>:/Icons/RotateLeft.png</normaloff>:/Icons/RotateLeft.png</iconset>
   </property>
   <property name="text">
    <string>Поворот по широте влево</string>
   </property>
   <property name="toolTip">
    <string>Поворот по широте влево</string>
   </property>
  </action>
  <action name="actionMoveRight">
   <property name="icon">
    <iconset resource="LibZ.qrc">
     <normaloff>:/Icons/RotateRight.png</normaloff>:/Icons/RotateRight.png</iconset>
   </property>
   <property name="text">
    <string>Поворот по широте вправо</string>
   </property>
   <property name="toolTip">
    <string>Поворот по широте вправо</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="icon">
    <iconset resource="LibZ.qrc">
     <normaloff>:/Icons/RotateUp.png</normaloff>:/Icons/RotateUp.png</iconset>
   </property>
   <property name="text">
    <string>Поворот по долготе вверх</string>
   </property>
   <property name="toolTip">
    <string>Поворот по долготе вверх</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="icon">
    <iconset resource="LibZ.qrc">
     <normaloff>:/Icons/RotateDown.png</normaloff>:/Icons/RotateDown.png</iconset>
   </property>
   <property name="text">
    <string>Поворот по долготе вниз</string>
   </property>
   <property name="toolTip">
    <string>Поворот по долготе вниз</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../Lib/CoreUi/Ui.qrc"/>
  <include location="LibZ.qrc"/>
 </resources>
 <connections/>
</ui>
